server/helpers/logger.ts

```diff
@@ -45,23 +45,28 @@ const labelFormatter = winston.format.label({
   label
 })
 
+const fileLoggerOptions = {
```

Comment:
`const fileLoggerOptions: FileTransportOptions = {`

Reply:
Thank you @Chocobozzz, I searched a way of setting fields more properly and I didn't find this type. It's really better.